en/flight_controller/mro_x2.1.md

```diff
@@ -85,3 +85,15 @@ make mro_x21_default
 ## Schematics
 
 The board is documented on the mRo hardware repo: [x21_V2_schematic.pdf](https://github.com/mRoboticsIO/Hardware/blob/master/X2.1/Docs/x21_V2_schematic.pdf).
+
+
+## Serial Port Mapping
+
+UART | Device | Port
+--- | --- | ---
+UART1 | /dev/ttyS0 | IO debug ? 
```

Comment:
I copied most of this in from connector info above. Can you confirm UART1 and UART8?

Reply:
alas There is no  UART1 on an F4 it is USART1  
@pkocmoud the schematic uses the name or where it connects too. 

This is what I believe it to be:
ttyS0 CONFIG_STM32_USART1=y Debug of IO
ttyS1 CONFIG_STM32_USART2=y  SERIAL1
ttyS2 CONFIG_STM32_USART3=y TELEM2
ttyS3 CONFIG_STM32_UART4=y GPS/I2C
ttyS4 CONFIG_STM32_USART6=y PX4IO 
ttyS5 CONFIG_STM32_UART7=y  SERIAL5  Console
ttyS6 CONFIG_STM32_UART8=y  SERIAL4

Reply:
I agree the schematic is confusing with the port naming. I will get that corrected. I agree with David's list.

Reply:
Thanks. Fixed up this bit and the connector mapping above in https://github.com/PX4/px4_user_guide/pull/672/commits/950ef64af22bf51faac220e50827a8b197978755